### ISSUE TITLE
About menu bouton

### ISSUE THREAD

**Issue by chronoss09** [Hydr8gon/NooDS#98]
Hi !!!

I like this emulator but not good for gba rom for the touch "L", this touch is for menu when you launch the game but you can't play some game without the touch L so, you need to change this bouton to "-" (minus), it will be better...

Sorry for my poor english :(

**Comment by Hydr8gon** [comment]
Sorry, but I don't understand the issue. None of the emulator's touch buttons should open the menu; the Android back button should do that, and nothing else. If the L button is opening the menu, then I have no idea why.

(Also sorry for taking so long to respond, I'm a mess when it comes to issues)

**Comment by chronoss09** [comment]
On the switch, when your press L or R buttons, it open the menu and for some game like Aladdin GBA, with th R bouton, he opens the sail so you can't because this bouton open the menu...

**Comment by Hydr8gon** [comment]
Oh, I understand now. The ZL and ZR buttons on the Switch should work as L and R buttons for the game.

**Comment by chronoss09** [comment]
Yes, do it with minus "-" boutton if possibl for the switch and Start or Select on the PSVita !!!

**Comment by Hydr8gon** [comment]
I can't do that, because start and select are GBA/DS buttons that games might use. The Switch port uses L and R because these buttons aren't used for anything else. I mapped the GBA/DS L and R buttons to ZL and ZR because these are bigger and that makes more sense to me. In the future I'll make the buttons remappable, so everything including the menu button can be changed.

**Comment by chronoss09** [comment]
Ok Good and thanks alot... I like your emu, its very good and work very well !!!
